### PR TITLE
BZ-1795572: Updating output to show new role binding created

### PR DESCRIPTION
**Pull request by bergerhoffer**
https://bugzilla.redhat.com/show_bug.cgi?id=1795572

Preview: http://file.rdu.redhat.com/~ahoffer/2019/BZ-1795572/admin_guide/manage_rbac.html#managing-role-bindings (internal)

### CODE REVIEW

**Comment by bergerhoffer** [comment]
@stlaz Can you review this update to properly show that a new role binding is created unless you specify it when adding the role?

Preview: http://file.rdu.redhat.com/~ahoffer/2019/BZ-1795572/admin_guide/manage_rbac.html#managing-role-bindings

File: admin_guide/manage_rbac.adoc [hunk 1]
```diff
@@ -712,8 +724,8 @@ Subjects:
   ----	----					---------
   Group	system:serviceaccounts:joe-project
 ----
-<1> The *alice* user has been added to the *admins* `*RoleBinding*`.
-
+<1> A new role binding is created with a default name, incremented as necessary. To specify an existing role binding to modify, use the `--rolebinding-name` option when adding the role to the user.
+<2> The user *alice* has been added.
```
Comment (ahardin-rh) [inline admin_guide/manage_rbac.adoc]: s/has been/is now

**Comment by ahardin-rh** [comment]
Just one suggestion from me. Looks great!

**Comment by bergerhoffer** [comment]
@barleyer Can you review this please?

**Pull request merged**